src/app/models/user.model.ts

```diff
@@ -0,0 +1,11 @@
+import { ResourceModel } from './resource.model';
+
+export class User extends ResourceModel<User> {
+  public name!: string;
```

Comment:
I don't like exclamation points here, it would be better to disable corresponding rule. Please discuss with a team